testpar/t_subfiling_vfd.c

```diff
@@ -438,16 +1064,14 @@ test_config_file(void)
         substr = strstr(config_buf, "hdf5_file");
         VRFY(substr, "strstr succeeded");
 
-        snprintf(scan_format, sizeof(scan_format), "hdf5_file=%%%zus", (size_t)(PATH_MAX - 1));
-        VRFY((sscanf(substr, scan_format, tmp_buf) == 1), "sscanf succeeded");
+        VRFY((sscanf(substr, "hdf5_file=%4095s", tmp_buf) == 1), "sscanf succeeded");
```

Comment:
We don't know how large PATH_MAX is, so using a constant like this could be problematic. That's why the previous version printed to a `scan_format` buffer first (same as below).